app/components/bookings/create/create-booking.component.html

```diff
@@ -0,0 +1,39 @@
+<ion-header>
+  <ion-navbar>
+    <ion-title>{{ _workpackage.stringId }} - {{ _workpackage.name }}</ion-title>
+  </ion-navbar>
+</ion-header>
+
+<ion-content>
+    <button accent floating block full *ngIf="!_bookingSession" (click)="createBookingSession()" >
+      <ion-icon name="ios-stopwatch-outline"></ion-icon>Livebuchung starten
+    </button>
+    <button accent floating block full *ngIf="_bookingSession" (click)="checkoutBookingSession()" >
```

Comment:
@marcelhub: The currently running `BookingSession` should only be parsed to the `CreateBookingComponent`, when the `CreateBookingComponent` was opened with the intention to close the booking session! That way you're still able to create a `Booking` completely independent of the running `BookingSession`, if it isn't parsed to the `CreateBookingComponent`.